.github/workflows/java.yml

```diff
@@ -51,9 +50,19 @@ jobs:
           mvn spotless:check
           popd
 
+      - name: Build and Install cpp
```

Comment:
You can change the gar version to v0.10.0, https://github.com/alibaba/GraphAr/blob/77f547ac14a786b6f557456cec3bb17e19256602/java/cmake/graphar-cpp.cmake#L51

no need to install latest cpp, it may make cpp update break the java ci